roles/wildfly_firewalld/meta/argument_specs.yml

```diff
@@ -0,0 +1,20 @@
+argument_specs:
+    main:
+        options:
+            wildfly_firewalld_package_name:
+                default: "firewalld"
+                description: "Package name for firewalld"
+                type: "str"
+            wildfly_app_firewalld_enabled:
+                default: "false"
+                description: "Indicate if firewalld is enabled for this collection"
+                type: "str"
+            wildfly_listen_ports:
+                default:
+                  - 8009
+                  - 8080
+                  - 8443
+                  - 4712
+                  - 4713
```

Comment:
Can you add 7600 which is the default jgroups-tcp port?

Reply:
The idea would be that if you don't use standalone, but standalone_full.xml, you'll need to provide the ports.

Maybe we can have a standalone_ports: var and a standalone_full_ports: with some sort of switch / if to select the right right set of ports depending on the config?

Thoughts?

Reply:
no it's ok; i didn't notice the jgroup-tcp socket binding is only defined in standalone-ha!